content/physical_properties/physical_properties_density.rst

```diff
@@ -0,0 +1,61 @@
+.. _physical_properties_density:
+
+Density
+*******
+
+Some Definitions
+================
+
+For gravity survey, the physical property of concern is density. A densier buried object would create an higher attraction than the background. A hole, with a density of 0, would create a lower attraction than the background. Difference of densities in the subsurface can be mapped by measuring the variations of the earth gravity field.
+
+Density is defined as the ratio between the mass m of an object and its volume V.
+
+.. math::											
+	\rho = \frac{m}{V} \quad &&\textrm{Volumetric mass density,}\ (g/cm^3)\ or \ (kg/m^3)
+
+
+Densities of geologic materials vary from 880 :math:`kg/m^3` (ice) (or 0 for air) to over 8000 :math:`kg/m^3` for some rare minerals. Rocks are generally between 1600 :math:`kg/m^3` (sediments) and 3500 :math:`kg/m^3` (gabbro). Table 2.1 from PV Sharma is reproduced below.
```

Comment:
instead of `880 :math: `kg/m^3``, it will read easier if you do `\\(880 ~\\text{kg/m}^3\\)` to render the number as math and not italicize the units